src/qtiCommonRenderer/renderers/interactions/InlineChoiceInteraction.js

```diff
@@ -58,7 +58,7 @@ var render = function(interaction, options) {
     _.extend(opts, options);
 
     if (opts.allowEmpty && !required) {
-        $container.find('option[value=' + _emptyValue + ']').text('--- ' + __('leave empty') + ' ---');
+        $container.find('option[value=' + _emptyValue + ']').text(__('--- leave empty ---') || '--- ' + __('leave empty') + ' ---');
```

Comment:
This line is looking a bit ugly. I believe there may be another way:

> Otherwise is there a way to pass a emptyLabelText option from customer extension to interaction?

There should be. If you add an entry in `config/tao/client_lib_config_registry.conf.php`:
```
        'taoQtiItem/qtiCommonRenderer/renderers/interactions/InlineChoiceInteraction' => array(
            'emptyLabelText' => 'foobar'
        ),
```
then in this JS file you can access it via:
```
import module from 'module';
console.log(module.config().emptyLabelText);
```
And merge it with the `_defaultOptions`, to allow overriding those too.

It would also be possible to add the `emptyLabelText` to the `render()` options, loaded from [config.js](https://github.com/oat-sa/tao-item-runner-qti-fe/blob/4730a594519aa5e32b2fd41975551a6ef8be6704/src/qtiCommonRenderer/renderers/config.js#L114), but this looks like more general options, so the specific one may be better.

Then the customer extension will require an install script + migration to set the config entry. Which is a bit more work, but looks like a more correct approach for this annoying piece of text.

Reply:
You are right. I was thinking only in the context of a monolingual customer, who would not mind configuring an unlocalised string. But the proposal would be unhelpful to other customers running multiple locale tests on a single instance.

I guess it's acceptable to keep this 1 line with the different variants of `--- leave empty ---`, for cases where the flexibility is needed.